src/odemis/driver/test/actuator_test.py

```diff
@@ -548,6 +546,67 @@ def test_metadata(self):
 
         testing.assert_pos_almost_equal(stage.speed.value, {"x": 10e-6, "y": 20e-6})
 
+    def test_update_axes_range(self):
+        """
+        Test the range calculation in ConvertStage
+        """
+        [a_min, a_max] = [0.e-3, 100.e-3]
+        [b_min, b_max] = [0.e-3, 13.e-3]
```

Comment:
This is more "pythonic":
```suggestion
        a_min, a_max = (0.e-3, 100.e-3)
        b_min, b_max = (0.e-3, 13.e-3
```